apps/core/src/hooks/useNftDetails.ts

```diff
@@ -64,9 +68,11 @@ export function useNftDetails(nftId: string, accountAddress: string | null) {
             if (value.includes('http')) {
```

Comment:
We just need the link to the owner, which is an explorer link, the rest can be removed, so I think removing this condition might solve the issue, right? that way, you wouldn't need to add the `disableLink` and `truncateLink` args to this function :pray:

Reply:
mmm im not understanding this function actually, how does `formatAddress` work in this context if value includes http? I might be wrong but seems off

Also, instead of adding `disableLink` I would rather just if it its a link never render a link and just truncate the value as it was before

Reply:
But the user won't be able to copy/paste the value (manually) if it's truncated, no ?